src/Lexeme.ts

```diff
@@ -58,6 +58,9 @@ export enum Lexeme {
     EndWhile,
     Eval,
     Exit,
+    // ExitFor isn't listed as a reserved word, but it seems like it'd match 'exitwhile'.
+    // TODO: Verify its reservedness on-device.
```

Comment:
Experimentally, `exitfor` isn't a reserved word, and `exit for` is the only way to exit a `for` loop:

```brightscript
for i=0 to 10
    print i
    if i = 2 then exit for
end for

exitfor = "not reserved"
print exitfor             ' prints "not reserved"
```

But `exitwhile` is indeed a reserved word, and is substitutable for `exit while`:

```brightscript
a = 0
while a < 10
    a = a + 1
    print a
    if a = 3 then exitwhile
end while

exitwhile = "not reserved?"    ' ERRNO: 175 - Exit While is not inside a While.
print exitwhile
```